validate.lic

```diff
@@ -766,6 +766,20 @@ class DRYamlValidator
             .each { |skill, _cooldown| error("caravan_training_skills: #{skill} is included, but no corresponding spell in crafting_training_spells!") }
   end
 
+  def assert_that_avtalia_array_is_valid(settings)
+    return if settings.avtalia_array.empty?
+
+    settings.avtalia_array.each do |camb|
+      first = camb['name'].split.first
+      second = camb['name'].split.last
+      short_reg = first == second ? /\b#{first}/i : /#{first}.*\b#{second}/i
+      error("avtalia_array appears to include your normal casting cambrinth.  This could break it horribly.") if short_reg =~ settings.cambrinth
+      if settings.avtalia_array.select { |camb| short_reg =~ camb['name'] }.to_a.length > 1
```

Comment:
❓ At a glance, it looks like this would always trigger because it would find its own entry. Can you confirm this does not trigger in the unique case and does trigger in the duplicate name case?

Reply:
Yes, you're right, except for the .to_a.length > 1.  My entries or a blank entry generates no error, but adding an entry for, say, 'tatt rob' generates an error re: uniqueness conflict with 'tattered robe.'